### PR TITLE
[Feat] Add a light theme to KeplerGl Prop

### PR DESCRIPTION
**Pull request by heshan0131**
- Allow passing `theme: 'light'` to `KeplerGl` component

Screen Shot 2019-04-22 at 6 21 52 PM

### CODE REVIEW

**Comment by heshan0131** [comment]
#339 #285 #346

File: README.md [hunk 1]
```diff
@@ -214,11 +214,11 @@ By Parsing `mint: false` kepler.gl will keep the component state in the store ev
 
 Read more about [Components][components].
 
-##### `theme` (Object, optional)
+##### `theme` (Object | String, optional)
 
 - default: `null`
 
-Object used to customize Kepler.gl style. Kepler.gl will use the value passed as input to override values from [theme](https://github.com/uber/kepler.gl/blob/master/src/styles/base.js).
+You can pass theme name or object used to customize Kepler.gl style. Kepler.gl provide an `'light'` theme besides the default 'dark' theme. When pass in a theme object Kepler.gl will use the value passed as input to override values from [theme](https://github.com/uber/kepler.gl/blob/master/src/styles/base.js).
```
Comment (macrigiuseppe) [inline README.md]: do you think we should provide a list of possible values? or a location in the code/docs where they can find values

File: src/constants/default-settings.js [hunk 2]
```diff
@@ -105,6 +105,11 @@ export const DIMENSIONS = {
   }
 };
 
+export const THEME = keyMirror({
```
Comment (macrigiuseppe) [inline src/constants/default-settings.js]: a think of description here will help people to understand they can use these values as kepler prop

**Comment by macrigiuseppe** [comment]
Nice one:
Screen Shot 2019-04-23 at 10 49 48 AM

File: README.md (hunk 1 above)
Comment (heshan0131) [reply README.md]: Hmm yea, we should definitely better document this. I am hoping the variable name itself should be self-explanatory.

File: src/constants/default-settings.js (hunk 2 above)
Comment (heshan0131) [reply src/constants/default-settings.js]: done

**Pull request merged**